v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/bigtablechangestreamstopubsub/BigtableChangeStreamsToPubSub.java

```diff
@@ -572,17 +579,45 @@ private static PubsubMessage createTestChangeMessage(PubSubUtils pubSub) throws
    * format.
    */
   static class ChangeStreamMutationToModJsonFn extends DoFn<ChangeStreamMutation, String> {
+    public static final String ANY_COLUMN_FAMILY = "*";
 
     private final BigtableSource sourceInfo;
+    private final Map<String, Set<String>> ignoredColumnsMap;
 
     ChangeStreamMutationToModJsonFn(BigtableSource source) {
       this.sourceInfo = source;
+
+      ignoredColumnsMap = new HashMap<>();
```

Comment:
Since this is shared across templates, should we pull this into BigtableSource and re-use that rather than duplicating it?